### PR TITLE
Added scroll table for desktop

### PR DESCRIPTION
**Pull request by NejcZdovc**
Based on #911 I added basic table css for desktop version. 

Because the comparison table is just to big I added option to display scroll for table on desktop and then mobile version for mobile devices.

### CODE REVIEW

**Comment by skipjack** [comment]
@NejcZdovc aside from my two review comments I did test and this does seem to resolve the issue from #911.

**Comment by NejcZdovc** [comment]
@skipjack I don't see your review comments

**Comment by skipjack** [comment]
@NejcZdovc sorry looks like I forgot to submit, you should be able to see them now.

File: content/guides/comparison.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ contributors:
 
 webpack is not the only module bundler out there. If you are choosing between using webpack or any of the bundlers below, here is a feature-by-feature comparison on how webpack fares against the current competition.
 
+<div class="scroll-table">
```
Comment (NejcZdovc) [inline content/guides/comparison.md]: Yes we can remove that and scroller to all tables on desktop.

File: styles/markdown.scss [hunk 2]
```diff
@@ -121,11 +121,23 @@
     }
   }
 
+  .scroll-table {
+    @include break {
+      overflow-x: auto;
```
Comment (NejcZdovc) [inline styles/markdown.scss]: Answered above.

File: content/guides/comparison.md (hunk 1 above)
Comment (skipjack) [inline content/guides/comparison.md]: Does it change anything to apply it to all? I'm not saying this is necessarily the best approach, it just seems like applying the same styles everywhere wouldn't change or hurt the styling of the other tables.

Comment (NejcZdovc) [inline content/guides/comparison.md]: I think it will not change anything. But I would need to test it out to be sure

Comment (skipjack) [inline content/guides/comparison.md]: Ok let me know once you've pushed the changes, I'll test again and we should be good to go.

**Comment by NejcZdovc** [comment]
@skipjack added overflow to all tables for desktop and looks good

**Pull request merged**